src/status_im/chat/models/message.cljs

```diff
@@ -107,9 +107,11 @@
 
 (defn- earlier-than-deleted-at?
   [{:keys [db]} {:keys [chat-id clock-value]}]
-  (let [{:keys [deleted-at-clock-value]}
+  (let [{:keys [deleted-at-clock-value public?]}
```

Comment:
Maybe we can leave only one function since they are identical?

Reply:
yep we definitely can

Reply:
fixed